package.json

```diff
@@ -2,21 +2,23 @@
   "name": "angular-feincms",
   "dependencies": {},
   "devDependencies": {
-    "grunt": "0.4.5",
-    "grunt-angular-templates": "0.5.7",
-    "grunt-contrib-concat": "1.0.0",
+    "grunt": "1.0.1",
+    "grunt-angular-templates": "1.0.4",
+    "grunt-contrib-concat": "1.0.1",
     "grunt-contrib-uglify": "1.0.1",
-    "grunt-contrib-watch": "0.6.1",
+    "grunt-contrib-watch": "1.0.0",
     "grunt-eslint": "18.1.0",
-    "grunt-jscs": "1.2.0",
-    "jit-grunt": "0.9.1",
-    "load-grunt-tasks": "3.3.0",
-    "time-grunt": "1.2.1"
+    "grunt-force-task": "2.0.0",
+    "grunt-jscs": "3.0.1",
+    "jit-grunt": "0.10.0",
+    "jquery": "3.0.0",
+    "load-grunt-tasks": "3.5.0",
+    "time-grunt": "1.3.0"
   },
   "engines": {
     "node": ">=0.10.0"
   },
   "scripts": {
-    "test": "grunt test"
+    "test": "grunt lint"
```

Comment:
Same as https://github.com/incuna/angular-feincms/pull/15/files/1db2da3fc5b7d4e52e7946bc72604c726896fddb..55a65145484c1f37be8a9c426814a53d7ea7a81f#r69094740: `grunt test` is better